index.js

```diff
@@ -0,0 +1,3 @@
+const {startServer} = require('./src/server')
+
+startServer(3052)
```

Comment:
`startServer( process.env.PORT || 3052 )` -> somebody may want to run your app on a different port.